src/components/ArticleToc.js

```diff
@@ -6,19 +6,8 @@ import { ListView } from 'components'
 export const ArticleToc = ({ items, close }) => {
```

Comment:
I would also like to rename the close function to be `closeFn`, same as the audio popup, I will do it in my later bug fix pr.